### PR TITLE
Hec 1398 lib upgrade

### PR DESCRIPTION
**Pull request by priyankadhiman3006**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,18 +2,18 @@ import play.core.PlayVersion
...
   val test = Seq(
-    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % "5.14.0"            % Test,
-    "org.scalatest"          %% "scalatest"              % "3.2.5"             % Test,
-    "org.jsoup"               % "jsoup"                  % "1.13.1"            % Test,
+    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % bootStrapVersion    % Test,
+    "org.scalatest"          %% "scalatest"              % "3.2.10"            % Test,
+    "org.jsoup"               % "jsoup"                  % "1.14.3"            % Test,
     "com.typesafe.play"      %% "play-test"              % PlayVersion.current % Test,
-    "com.vladsch.flexmark"    % "flexmark-all"           % "0.36.8"            % "test, it",
+    "com.vladsch.flexmark"    % "flexmark-all"           % "0.62.2"            % "test, it",
```
Comment (AndyHWChung) [inline project/AppDependencies.scala:16]: looks like there's might be some sort of problem with this looking at the PR builder failed output

Comment (priyankadhiman3006) [reply project/AppDependencies.scala:16]: yup same like last time. reverting it

**Comment by priyankadhiman3006** [comment]
@AndyHWChung @meghana-viswanath - build has passed now

File: project/AppDependencies.scala (hunk 1 above)
Comment (meghana-viswanath) [reply project/AppDependencies.scala]: sorry, didn't quite catch what the issue was here? is it version incompatibility?

Comment (priyankadhiman3006) [reply project/AppDependencies.scala]: this version
"com.vladsch.flexmark"    % "flexmark-all"           % "0.62.2"   
seems to be incompatible with others, so reverted it back

**Review by meghana-viswanath — APPROVED** [review]

File: project/AppDependencies.scala (hunk 1 above)
Comment (priyankadhiman3006) [reply project/AppDependencies.scala]: @AndyHWChung , could you please resolve and approve this, if everything seems ok to you?

Comment (AndyHWChung) [reply project/AppDependencies.scala]: sorry, missed this :see_no_evil:  looks good :+1:

**Review by AndyHWChung — APPROVED** [review]

**Pull request merged**